src/limbo/bayes_opt/bo_base.hpp

```diff
@@ -94,45 +93,39 @@ namespace limbo {
 
     namespace bayes_opt {
 
-        typedef boost::parameter::parameters<boost::parameter::optional<tag::acquiopt>,
-            boost::parameter::optional<tag::statsfun>,
+        typedef boost::parameter::parameters<boost::parameter::optional<tag::statsfun>,
             boost::parameter::optional<tag::initfun>,
             boost::parameter::optional<tag::acquifun>,
```

Comment:
We said, but right now I am using acquisition functions in IMGPO (it could need some cleaning - but we still have it in experimental). Check [here](https://github.com/resibots/limbo/blob/f78f6fea62053bd24e1555905cd952f5f8943b02/src/limbo/experimental/bayes_opt/imgpo.hpp#L160).